src/store/plugins/names.js

```diff
@@ -72,8 +76,12 @@ export default store =>
                 { root: true },
               );
             }
+            if (name.expiresAt <= height && oldName.autoExtend) {
```

Comment:
I think the logic here is wrong. 

If the `height` is gt than `name.expiresAt` then the name would have been already expired. 🤔 

Maybe just auto extend without checking the height?